### DISCUSSION TITLE
GraphQL batchOperations Feature Request

### DISCUSSION THREAD

**Discussion by Nosferican** [community/community#24574] [Apps, API and Webhooks]
<p>Would it be possible to request this feature (reference <a href="https://youtu.be/ViXL0YQnioU" rel="nofollow noopener">https://youtu.be/ViXL0YQnioU</a>)?</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/1X/50828cde0df753ef5f53256490b5f2a290c7d179.png" data-download-href="/uploads/short-url/budTOH8ammzjU5Yv4xEciYIoGMN.png?dl=1" title="b940c9f6-5fdf-11e6-8d31-9baee04e70ff" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/2529329/181085299-bd91763d-764e-4893-8a3c-44eb37592e2c.png" alt="b940c9f6-5fdf-11e6-8d31-9baee04e70ff" data-base62-sha1="budTOH8ammzjU5Yv4xEciYIoGMN" width="690" height="385" srcset="https://aws1.discourse-cdn.com/github/optimized/1X/50828cde0df753ef5f53256490b5f2a290c7d179_2_690x385.png, https://aws1.discourse-cdn.com/github/original/1X/50828cde0df753ef5f53256490b5f2a290c7d179.png 1.5x, https://aws1.discourse-cdn.com/github/original/1X/50828cde0df753ef5f53256490b5f2a290c7d179.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/1X/50828cde0df753ef5f53256490b5f2a290c7d179_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">b940c9f6-5fdf-11e6-8d31-9baee04e70ff</span><span class="informations">775×433 53.6 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by logankilpatrick** [comment, marked as answer]
<p>Hey <a class="mention" href="/nosferican">@nosferican</a>, all feature request should go directly to GitHub support via email or through this feedback form: <a href="https://support.github.com/contact/feedback" rel="nofollow noopener">https://support.github.com/contact/feedback</a></p>
